sdcm/utils/data_validator.py

```diff
@@ -790,6 +790,8 @@ def validate_deleted_rows(self, session, during_nemesis=False):
                 ).publish()
             else:
                 LOGGER.debug('Validation deleted rows finished successfully')
+        elif len(actual_result) == self.rows_before_deletion:
```

Comment:
Probably may be useful to create a `warning event`?

Reply:
it's equivalent, no one would read any of them.

I'm guessing this is intend to whom has a deletion case, and assume he'll read the log/events to look it up

I would argue that the vaildator need some configuration, so the test case can set which validation to do, and which are not needed.

Reply:
A debug message changed to event